documentation/staging/content/faq/newbie.md

```diff
@@ -0,0 +1,139 @@
+
+For T3 communication outside the Kubernetes cluster, we use NodePort and configure a WebLogic channel for T3 RMI.  See the blog, [T3 RMI Communication for WebLogic Server Running on Kubernetes](https://blogs.oracle.com/weblogicserver/t3-rmi-communication-for-weblogic-server-running-on-kubernetes), which explains RMI T3 communication in detail. All QoS are supported for T3 and HTTP communication (SSL, administration port).
+
+**Q:** Are clusters supported on Kubernetes using both multicast and unicast?
+
+**A:** Only unicast is supported. Most Kubernetes network fabrics do not support multicast communication. Weave claims to support multicast but it is a commercial network fabric.  We have certified only on Flannel, which supports unicast only.
+
+
+**Q:** Docker resources: We use an SSL listener, SSL port, and an administration port on each instance. How do we map these resources?
+
+**A:** Yes, you can use an SSL listener, SSL port, and an administration port in both Docker and Kubernetes.  See to the blog, [T3 RMI Communication for WebLogic Server Running on Kubernetes](https://blogs.oracle.com/weblogicserver/t3-rmi-communication-for-weblogic-server-running-on-kubernetes), which describes how to map these ports in Kubernetes.
+
+***
+
+#### Transactions
+
+**Q:** Are XA transactions and recovery also supported?
+
+**A:** Yes, XA transactions are supported. We are expanding our certification to include more complex cross-domain XA transaction use cases.
+
+***
+
+**Q:** For binding EJBs (presentation/business-tier), are unique and/or dynamic domain-names used?
+
+**A:** We do not enforce unique domain names. If you deploy two domains that must interoperate using RMI/EJB/JMS/JTA/and such, or that share RMI/EJB/JMS/JTA/and such clients, which concurrently communicate with both domains, then, as usual, the domain names must be configured to be different (even if they have different `domain-uids`).
+
+***
+
+#### Load Balancers
+
+**Q:** Load balancing and failover inside a DataCenter (HTTPS and T3s)?
+
+**A:** We originally certified on Traefik with the Kubernetes cluster; this is a very basic load balancer.  We have also certified other more sophisticated load balancers. See [Ingress]({{< relref "/userguide/managing-domains/ingress/_index.md" >}}).
+
+***
+
+#### Life Cycle and Scaling
+
+**Q:** How to deal with grow and shrink? Cluster and non-cluster mode.
+
+**A:** You can scale and shrink a configured WebLogic cluster (a set of preconfigured Managed Servers) or a dynamic WebLogic cluster (a cluster that uses templated Managed Servers) using different methods. See the blog, [Automatic Scaling of WebLogic Clusters on Kubernetes](https://blogs.oracle.com/weblogicserver/automatic-scaling-of-weblogic-clusters-on-kubernetes-v2).
```

Comment:
Is this blog up to date? It's dated 2017.

Reply:
Not sure. We can ask @lennyphan

Reply:
This blog is out of date.  The WLDF description is mostly valid for the rules but the use of webhooks pre-dates the operator's REST API.

Reply:
Thanks for the review. I'll remove the reference and, perhaps, point them to the Scaling section in the docs, https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle/scaling/ Does that work for you?

Reply:
Yes, this is much better link.  Thank you!

Reply:
Done.